source/scripts/ev3dev-powerinfo

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+
+print_val() {
+  printf "%-20s%s\n" "$1: " "$2"
+}
+
+print_power_info() {
+    for pwr_supply in /sys/class/power_supply/*; do
+        uVolts=`cat $pwr_supply/voltage_now`
+        intVolts=`expr $uVolts / 1000000`
+        decmicroVolts=`expr $uVolts - $intVolts \* 1000000`
+        decmiliVolts=`expr $decmicroVolts / 1000`
+        printf "%-20s: %3d.%03d  V\n" "Voltage" $intVolts $decmiliVolts
+
+
+        uAmps=`cat $pwr_supply/current_now`
+        intmiliAmps=`expr $uAmps / 1000`
+        decmiliAmps=`expr $uAmps - $intmiliAmps \* 1000`
+        printf "%-20s: %3d.%03d mA\n" "Current" $intmiliAmps $decmiliAmps
+
+
+        miliVolts=`expr $intVolts \* 1000 + $decmiliVolts`
+        miliAmps=`expr $intmiliAmps`
+        uWatts=`expr $miliVolts \* $miliAmps`
+        intWatts=`expr $uWatts / 1000000`
+        decmicroWatts=`expr $uWatts - $intWatts \* 1000000`
+        decmiliWatts=`expr $decmicroWatts / 1000`
+        printf "%-20s: %3d.%03d  W\n" "Power" $intWatts $decmiliWatts
+    done
+}
+
+
+print_val "Power Supply"
+print_power_info
+
+printf "\n"
+print_val "/sys/class/power_supply info"
+for pwr_supply in /sys/class/power_supply/*; do
+    cat $pwr_supply/uevent
```

Comment:
Isn't this a bit redundant?

Reply:
I  wasn't sure that my unit conversion algorithm was correct when I first wrote it. In addition, I thought some of the raw info may be relevant but was too lazy to convert it to a human readable format.

Reply:
maybe we just keep it super simple and make the whole script something like this:

```
for pwr_supply in /sys/class/power_supply/*; do
    echo "Power supply: $(basename $pwr_supply)"
    cat $pwr_supply/uevent
done